frontend/src/Button.css

```diff
@@ -0,0 +1,7 @@
+.Button {
```

Comment:
Awesome! That seems like a great pattern!

Can you think of any times where it might be painting ourselves into a corner? Maybe with nested components?